features/tags_and_wrangling/tag_fandom_relationships.feature

```diff
@@ -0,0 +1,8 @@
+Feature: Fandom Relationship Tags Link
+
+  Scenario: Click on "Relationships by Character" on a fandom tag containing periods  
+    Given a canonical fandom "Harry Potter - J. K. Rowling"
+    When I view the tag "Harry Potter - J. K. Rowling"
+        And I follow "Relationship tags in this fandom" 
+    Then I should not see "The page you were looking for doesn't exist."
+        And I should see "Relationships by Character"
```

Comment:
I was pretty sure we already had a test for this link and didn't need a new feature file, so I tracked the existing test down: it's in `features/other_b/fandoms.feature` (added in #4350). Could you move this test to that file instead? Then I think this will be all done!